gmt-web-app/package.json

```
@@ -0,0 +1,75 @@
+{
+    "name": "gmt-web-app",
+    "version": "0.1.0",
+    "private": true,
+    "type": "module",
+    "testEnvironment": "jsdom",
+    "dependencies": {
+        "@fortawesome/fontawesome-svg-core": "^6.5.1",
+        "@fortawesome/free-solid-svg-icons": "^6.5.1",
+        "@fortawesome/react-fontawesome": "^0.2.0",
+        "@testing-library/jest-dom": "^5.17.0",
+        "@testing-library/react": "^13.4.0",
+        "@testing-library/user-event": "^13.5.0",
+        "@types/jest": "^27.5.2",
+        "@types/node": "^16.18.68",
+        "@types/react": "^18.2.45",
+        "@types/react-dom": "^18.2.17",
+        "axios": "^1.6.2",
+        "nth-check": "^2.1.1",
+        "postcss": "^8.4.32",
+        "react": "^18.2.0",
+        "react-dom": "^18.2.0",
+        "react-router-dom": "^6.21.0",
+        "react-scripts": "^5.0.1",
+        "resolve-url-loader": "^5.0.0",
+        "styled-components": "^6.1.1",
+        "web-vitals": "^2.1.4"
+    },
+    "scripts": {
+        "start": "react-scripts start",
+        "build": "react-scripts build",
+        "test": "react-scripts test",
+        "eject": "react-scripts eject",
+        "lint": "eslint . --ext .ts,.tsx",
+        "format": "prettier . --write"
```

As mentionned above, we need a check and a write version for both of these. (write for lint is just bonus but I like to have it)

Something like:
```suggestion
        "lint": "eslint . --ext .ts,.tsx",
        "lint:fix": "eslint . --ext .ts,.tsx --fix",
        "format": "prettier . --write",
        "format:check": "prettier ."
```